src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASKS = 100;
+    public static final String BORDER_DECORATION = "_____________________________________________________";
```

Comment:
Perhaps consider making the constants `MAX_TASKS` and `BORDER_DECORATION` `private` or `protected`. Though these variables do not pose any immediate security issues, I feel that classes outside the scope of this project should not have direct access to the variables and hence, the access modifier should be reduced from `public` to either `protected` or `private`.